Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -298,25 +312,69 @@ theorem degrees_esymm [Nontrivial R] (n : ℕ) (hpos : 0 < n) (hn : n ≤ Fintyp
 
 end ElementarySymmetric
 
+section CompleteHomogeneousSymmetric
+
+open Finset Multiset Sym
+
+variable [DecidableEq σ] [DecidableEq τ]
+
+/-- The `n`th complete homogeneous symmetric `MvPolynomial σ R`. -/
```

Comment:
Please expand this docstring to include the informal definition of "complete homogeneous symmetric polynomial".